### PR TITLE
Added AddListingScope example.

### PR DESCRIPTION
**Pull request by fiboknacky**

### CODE REVIEW

File: examples/ShoppingAds/AddListingScope.php
Comment (PierrickVoulet) [inline examples/ShoppingAds/AddListingScope.php]: nit: "ListingScope" is a type, do we want to change for english words like "listing scope" to make it consistent with all code example comments?

File: examples/ShoppingAds/AddListingScope.php
Comment (PierrickVoulet) [inline examples/ShoppingAds/AddListingScope.php]: In the next major release, ListingBrandInfo will be renamed to ProductBrandInfo. Like the AddShoppingProductListingGroupTree code example, this will need to be fixed.

What about waiting for the next release before reworking/merging this code example?

File: examples/ShoppingAds/AddListingScope.php
Comment (PierrickVoulet) [inline examples/ShoppingAds/AddListingScope.php]: In our other code examples, the operation is created and stored in a dedicated variable. You might want to do the same here for consistency.

**Review by PierrickVoulet — CHANGES_REQUESTED** [review]

File: examples/ShoppingAds/AddListingScope.php
Comment (fiboknacky) [reply examples/ShoppingAds/AddListingScope.php]: Actually, this is more consistent with all other places, i.e., when creating an object, we now specify a field directly (like creating ListingDimensionInfo above).
We don't use `setProductType()` method, for instance.

Because of the above reason and its conciseness, I actually prefer this style.
But I just don't feel necessary to change all the examples to follow this.

I'll change this to match other examples for now, but please also let me know what you think in the long run. :)

Comment (PierrickVoulet) [reply examples/ShoppingAds/AddListingScope.php]: I prefer this one too: easily understandable, readable and more concise.

**Review by PierrickVoulet — APPROVED** [review]

**Review by jradcliff — APPROVED** [review]

**Pull request merged**
